src/modules/audio/spatializers/oculus_spatializer.c

```diff
@@ -0,0 +1,237 @@
+#include "audio/audio.h"
+#include "audio/spatializer.h"
+#include "lib/miniaudio/miniaudio.h"

```

Comment:
would prefer to not rely on miniaudio here.  it's used for ma_mutex, maybe tinycthread could be used?  hooray for mixing mutex implementations